### ISSUE TITLE
Not working on Yuzu Switch Emulator

### ISSUE THREAD

**Issue by ReaperOfSouls1909** [shiftinv/MotionSourceiOS#1]
https://github.com/yuzu-emu/yuzu/pull/3817 could you take a look please

**Comment by anirudhb** [comment]
This also occurs with WebGyroForCemuhook

**Comment by shiftinv** [comment]
After looking at the code in the PR I managed to get it to connect by adding `,host=<device ip>` to the `motion_device` config variable. `UDPMotionFactory::Create` only receives the `motion_device` params, looks like the `udp_input_address/_input_port/_pad_index` config variables are not used anymore.
Seems like it doesn't work ingame yet, even though it correctly receives the motion data:
![image](https://user-images.githubusercontent.com/8530778/80744628-453b9080-8b1f-11ea-8b73-a3b955aa38b9.png)

**Comment by anirudhb** [comment]
Yes, that's what the problem has been. However I've noticed that this only occurs with motionsource and such, and not on real hardware like ds4 and joycon :)

**Comment by AdminC21332** [comment]
I think the problem is because yuzu doesnt have the pad number 0 option. Because when i checked in padtest motionsourceios appeared as pad 0 and yuzu said it was working properly after clicking detect controllers was because it detected that pad 0 but the emulator didn't even know it existed. I'm not really sure
